lib/live_view_native/live_session.ex

```diff
@@ -8,10 +8,10 @@ defmodule LiveViewNative.LiveSession do
   def on_mount(:live_view_native, _params, _session, socket) do
     with %{"_platform" => platform_id} <- get_connect_params(socket),
          platforms <- LiveViewNative.platforms(),
-         %LiveViewNativePlatform.Context{} = platform_context <- Map.get(platforms, platform_id) do
+         %LiveViewNativePlatform.Context{} = platform_context <- Map.get(platforms, platform_id)  do
```

Comment:
Did you mean to add the extra space before `do`?

Reply:
Oops nope 😊 fixed!